src/actions/pause/mapScopes.js

```diff
@@ -291,6 +284,168 @@ async function findGeneratedBinding(
   };
 }
 
+async function findGeneratedBindingFromPosition(
```

Comment:
this might just be my style, but it would be easier to follow if we had a `utils/pause/mapScopes` which had a module `findGeneratedBindingFromPosition.js` so i can see what's needed for this algorithm and what's public and what's private to it

Reply:
So make `mapScopes` a folder with `index.js` and `findGeneratedBindingFromPosition.js`?

Reply:
yep